src/styles/main.scss

```diff
@@ -1,3 +1,29 @@
+@import "utils/variables";
+@import "utils/functions";
+@import "reset";
+
 body {
   margin: 0;
+  font-style: normal;
+  font-weight: 400;
+  font-family: Roboto, sans-serif;
+  overflow-x: hidden;
```

Comment:
There is no need for this property, it may be removed.

Reply:
I deleted `overflow-x`.